### DISCUSSION TITLE
Charticulator Visual Community Viewer Edition, v. 1.4.0.0 availability from Microsoft App Source

### DISCUSSION THREAD

**Discussion by SMusento23** [zBritva/charticulator-visual-community#38] [General]
When will Viewer edition, v. 1.4.0.0 be available for download from Microsoft App Source?

**Comment by zBritva** [comment]
The Editor 1.4.0.0 is approved. Should reach end users in 2weeks 

The View is under certification, usually takes long time (3-5weeks)
